src/components/MainPage/Products/GridButton/GridButton.tsx

```diff
@@ -0,0 +1,34 @@
+import './gridButton.scss';
+import { GridButtonType } from './types/gridButton.interface';
+
+function GridButton({
+  viewState, toggleView,
+}: GridButtonType) {
+  return (
+    <button
+      onClick={toggleView}
+      className={`products__buttons views__buttons ${viewState ? 'views__buttons_active' : ''}`}
+      type="button"
+    >
+      {viewState ? (
+        <svg width="24" height="24" viewBox="0 0 24 24">
```

Comment:
I don't really like the idea of putting a svg in the markup like that. Options with background or img look better for me.

Reply:
+
replace by component
the problem is that when pasting into the source, I will not be able to change the fill color